x-pack/plugins/task_manager/server/polling/task_poller.ts

```diff
@@ -112,6 +120,7 @@ export function createTaskPoller<T, H>({
     events$: subject,
     start: () => {
       if (!running) {
+        logger.info('Starting the task poller');
         running = true;
         runCycle().catch(() => {});
```

Comment:
maybe a lot inside the `catch`?

Reply:
Added in this commit https://github.com/elastic/kibana/pull/194741/commits/d808e20ebed23fdda2704736856f2abb70fa4adc